src/ConnectedMode/Migration/ConnectedModeMigration.cs

```diff
@@ -61,24 +61,25 @@ public async Task MigrateAsync(BoundSonarQubeProject oldBinding, IProgress<Migra
 
             logger.WriteLine(MigrationStrings.Starting);
 
-            // TODO: cleanup - dummy implementation to provide feedback for the UI
-            progress?.Report(new MigrationProgress(1, 2, "TODO 1", false));
-            progress?.Report(new MigrationProgress(2, 2, "TODO 2", true));
```

Comment:
I've added some more representative `progress?.Report` calls, but they are not supposed to be the final version.